gallery/src/components/NxTable/NxTablePaginationFilterExample.tsx

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, {useState} from 'react';
+
+import {
+  NxTable,
+  NxTableBody,
+  NxTableCell,
+  NxTableHead,
+  NxTableRow,
+  NxPagination,
+  NxFilterInput
+} from '@sonatype/react-shared-components';
+import { slice, toLower, pipe, prop, includes, filter, both } from 'ramda';
+
+interface Row { name: string; country: string };
+
+const PAGE_SIZE = 5;
```

Comment:
Is there a reason why `PAGE_SIZE` is all caps?

Reply:
> Is there a reason why PAGE_SIZE is all caps?

Tranditionally "constants" have been written this way. It's a bit wishy washy in the coding style I use since most variables are constants in some sense.  But I tend to still use ALL_CAPS for module-global primitive constants.